rootfs/Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV LANG en_US.utf8
 RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
 
 ENV PG_MAJOR 9.4
-ENV PG_VERSION 9.4.6-1.pgdg80+1
+ENV PG_VERSION 9.4.7-1.pgdg80+1
```

Comment:
#78 will be merged when CI goes consistently green again and make that PR go green.